src/plugins/generator/outputs/tags.js

```diff
@@ -39,7 +39,7 @@ export async function tags() {
 	progressBar.increment()
 
 	// Set project types
-	const projectTypes = ['mod', 'modpack']
+	const projectTypes = ['mod', 'modpack', 'plugin', 'resourcepack']
```

Comment:
Plugin isn't actually a project type on the backend. This isn't used to generate the header, is it?

Reply:
oh my bad i forgot about that, but yeah it's currently used to generate the nav headers on the rewrite

Reply:
with how the rewrite is set up, i'm not really sure how to add plugins without just explicitly setting it, other than adding this to the generator plugin